Learn/js/upperCase_script.js

```diff
@@ -0,0 +1,13 @@
+function upperCaseString() {
+    var str2 = document.getElementById("str2").value;
+    var res = 0;
+    var lgth = str2.length;
+    for (var i = 0; i < lgth; i++) {
+        if (Check(str2[i]) == false) {
```

Comment:
not an issue but if we rename function **Check**  to **isNumber** it will look more nicely 
```
if (!isNumber(str2[i])) {
...
}
```

Reply:
ok, i will fix

Reply:
done